client/src/js/agreement/CreateAgreementContainer.jsx

```diff
@@ -28,9 +25,45 @@ function CreateAgreementContainer () {
   const classes = useStyles()
   return (
     <Paper elevation={23} className={classes.root}>
-      <h2 className={classes.h2}>Sign a CLA</h2>
-      <p className={classes.textCenter}>Who will you be submitting contributions on behalf of?</p>
       <Grid container spacing={3} justify='center'>
+        <Grid item xs={12}>
+          <h2 className={classes.h2}>Sign a CLA</h2>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2" className={classes.textCenter}>
+                When to sign an Individual CLA?
+              </Typography>
+              <Typography variant="body2" component="div">
+                I should sign this CLA if:
+                <ul>
+                  <li>I'm making contributions
+                    on my own free time and no-one is contracting me for them.</li>
+                </ul>
+              </Typography>
+            </CardContent>
+          </Card>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2" className={classes.textCenter}>
+                When to sign an Institutional CLA?
+              </Typography>
+              <Typography variant="body2" component="div">
+                I should sign this CLA if:
+                <ul>
+                  <li>I'm making contributions on behalf of my employer</li>
```

Comment:
Someone is paying me to contribute to the project
I'm making contributions on behalf of my employer

In these circumstances, it is very likely that the person paying you to do the work owns the rights to it. They need to sign the CLA for these contributions.

Reply:
Is this what you were suggesting?
```
-                  <li>I'm making contributions on behalf of my employer</li>
-                  <li>I'm emplying (or contracting with) someone to contribute to the project</li>
+                  <li>Someone is paying me to contribute to the project</li>
+                  <li>I'm employing (or contracting with) someone to contribute to the project</li>
```

Do we need to add `In these circumstances, it is very likely that the person paying you to do the work owns the rights to it. They need to sign the CLA for these contributions.`?

The next page question (`Are you authorized to sign legal agreements for your company?` should cover that part

Reply:
"I'm making contributions as an employee or under a contract or other agreement with a company, and that company holds the copyright and patents rights on my contributions."